### DISCUSSION TITLE
Play sound when checking a task.

### DISCUSSION THREAD

**Discussion by Osiris-Team** [iSoron/uhabits#1192] [Feature Requests] [answered]
I would like to hear a satisfying "Ding!" or "Ping!" sound when checking a task.
I could implement it myself and write the code for it, if you don't want to.
So I wanted to ask if you'd accept my future pull request.

**Comment by hiqua** [comment, marked as answer]
Dup: #654

**Reply by Osiris-Team** [reply]
k gonna create a PR then, since that feature request is from 2020

**Reply by patrik-bartak** [reply]
Has this been followed up on? It seems not, yet it's quite a basic and satisfying feature. I'd be happy to contribute if necessary.

**Reply by Osiris-Team** [reply]
Yeah I dropped it since I got sidetracked by other things :/

**Reply by patrik-bartak** [reply]
And are you still too busy for it?

**Reply by Osiris-Team** [reply]
Yeah u go for it!
